flows/definition/dependencies.go

```diff
@@ -47,13 +47,8 @@ func (d *dependencies) refresh(sa flows.SessionAssets, missing assets.MissingCal
 	}
 	for i, ref := range d.Fields {
 		a := sa.Fields().Get(ref.Key)
-
 		if a == nil {
-			// TODO for now if a field reference came from an expression (i.e. no name), we don't blow up if it's missing
```

Comment:
since caller will now get back list of missing assets, it's up to them how strictly to deal with contact fields